main.py

```diff
@@ -2,9 +2,47 @@
 
 from db.models import Race, Skill, Player, Guild
 
+import json
+
```

Comment:
Change the order of your imports according to `pep8`. Here `json` should be first.